java/pattern/constraint/TypeConstraint.java

```diff
@@ -466,7 +466,22 @@ public boolean isOwns() {
 
         @Override
         public String toString() {
-            return OWNS.toString() + SPACE + attributeType + (isKey ? "" + SPACE + IS_KEY : "");
+            StringBuilder syntax = new StringBuilder();
+            syntax.append(OWNS).append(SPACE);
+
+            if (!attributeType.label().isPresent()) syntax.append(attributeType);
+            else syntax.append(attributeType.label().get().label());
```

Comment:
This logic is not needed given that `TypeVariable.toString()` already takes care of this. You can infer this from the fact that the previous code worked without this new logic.